### ISSUE TITLE
test failure in ByteVectorJvmTest

### ISSUE THREAD

**Issue by SethTisue** [scodec/scodec-bits#78]
seen in a local run of the Scala 2.12 community build:

```
[scodec-bits] [info] ByteVectorJvmTest:
...
[scodec-bits] [info] - gzip - nowrap *** FAILED *** (136 milliseconds)
[scodec-bits] [info]   TestFailedException was thrown during property evaluation.
[scodec-bits] [info]     Message: Left(java.util.zip.DataFormatException: Insufficient data -- inflation reached end of input without completing inflation - ByteVector(34 bytes, 0x69007f74c17fff00017f1001807f3700010c00017f7f00017f010cec57a08001757f)) was not equal to Right(ByteVector(35 bytes, 0x69007f74c17fff00017f1001807f3700010c00017f7f00017f010cec57a08001757f01))
[scodec-bits] [info]     Location: (ByteVectorJvmTest.scala:95)
[scodec-bits] [info]     Occurred when passed generated values (
[scodec-bits] [info]       arg0 = ByteVector(35 bytes, 0x69007f74c17fff00017f1001807f3700010c00017f7f00017f010cec57a08001757f01)
[scodec-bits] [info]     )
```

**Comment by SethTisue** [comment]
seen again today at https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/2183/consoleFull

**Comment by mpilquist** [comment]
Fixed in 58f96f60a340c48564ce2d578855d303a6ba16e6

**Comment by nadavwr** [comment]
We'$e seeing odd behavior where `.deflate()` (with nowrap disabled, as by default) results in files that are rejected by libraries trying to decompress them on other platforms. I see this test was just dropped instead of fixed -- could this be a longstanding issue that's being missed by tests?
